src/google-classroom-extractor/README.md

```diff
@@ -1,37 +1,38 @@
 # Google Classroom Extractor
 
-The purpose of this project is to make it easy to extract
-Google Classroom data in order to monitor the progress and
-participation of students in virtual or blended models.
-
-This data extractor targets two audiences.  The first is
-someone who just wants the data out of Google Classroom into a
-.csv file that can be loaded into a spreadsheet.  The second
-for someone who is familiar with Jupyter Notebook, a common
-Python-based tool for analysis and reporting.
-
-## Installing the Extractor
-
-1. Install a current version of Python.  The extractor has
-   been tested against Python 3.8.5.  You can find it
-   [here](https://www.python.org/downloads/).
-1. Install Poetry, a Python tool for downloading third-party
-   libraries.  You can find installation instructions
-   [here](https://python-poetry.org/docs/#installation). The
-   instructions are basically a long single-line command to
-   copy and run on Powershell (for Windows) or bash (for
-   Mac/Linux).
-1. Clone this GitHub repository and open a command prompt in
-   the root directory of this project (where this README file
-   resides).
-1. Configure Poetry to put Python enivronments in the right place: `poetry
-   config virtualenvs.in-project true`.
-1. Install the third-party libraries used by the extractor:
-   `poetry install`. This may take a while.
-1. Copy the "example.env" file to ".env" in the same directory
-   and set the variables as documented in the file.
-1. Place the service-account.json file downloaded earlier in
+This tool retrieves and writes out to CSV students, active sections,
+assignments, and submissions by querying the Google Classroom API. For more
+information on the this tool and its output files, please see the main
+repository [readme](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit).
+
+## Getting Started
+
+1. Ensure you have [Python 3.8+ and
+   Poetry](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit#getting-started)
+1. At a command prompt, install all required dependencies:
+
+   ```bash
+   poetry install
+   ```
+
+1. Optional: make a copy of the `.env.example` file, named simply `.env`, and
+   customize the settings as described in the Configuration section below.
+1. Place the service-account.json file described below into
    the root directory of this project.
+1. Run the extractor one of two ways:
+   * Execute the extractor with minimum command line arguments:
+
+      ```bash
+      poetry run python edfi_google_classroom_extractor -a [admin account email]
+      ```
+
+   * Alternately, run with environment variables or `.env` file:
+
+     ```bash
+     poetry run python edfi_google_classroom_extractor
+     ```
+
+   * For detailed help, execute `poetry run python canvas_extractor -h`.
 
 ## Running the Extractor
```

Comment:
This header text is not present in the Canvas nor Schoology readme files

Reply:
```suggestion
## Configuration
```